tests/general/InvalidLinkBearTest.py

```diff
@@ -316,7 +316,8 @@ def test_pip_vcs_url(self):
         bzr+http://httpbin.org/status/404#egg=coala-bears
         """.splitlines()
 
-        brokenlink_at_hash = """git+http://httpbin.org/status/404@master#egg=coala-bears
+        brokenlink_at_hash = """
```

Comment:
When invoking `splitlines`, this change produces an empty string at the beginning of the returned list. Does this affect behaviour?

I would actually recommend to just write up a list and don't invoke `splitlines` at all.

Reply:
No it doesnt.
Tests would fail.  See lines below which already use that behaviour.

Reply:
:+1: